project.clj

```diff
@@ -1,13 +1,12 @@
-(defproject clojurewerkz/cassaforte "3.0.0-alpha2-SNAPSHOT"
```

Comment:
Absolutely. Will change it back. This one was only for me to be able to distinguish a local copy from the clojars copy.